CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.2.0-dev
```

Comment:
should we go ahead and release now?

Reply:
We could. I'd prefer to wait until the discussion on the SDK CL settles so that we are 100% sure of the name.

Reply:
sgtm, it might be good to add a `publish_to: none` and an issue we can link there to track all of this?

Reply:
https://github.com/dart-lang/http_multi_server/issues/23